plugins/inputs/sysstat/README.md

```diff
@@ -236,7 +234,7 @@ If you change the group value to false like below:
 
 you get the following output:
 
-```shell
+```text
 $ telegraf -config telegraf.conf -input-filter sysstat -test
 * Plugin: sysstat, Collection 1
 > io_tps value=0.5 1459255780126025822
```

Comment:
Missed to completely fix

Reply:
Done. https://github.com/influxdata/telegraf/pull/12989/commits/8eb358f9252a7a5e4e9acef8b601556b722c608a